dnf-behave-tests/features/microdnf/install-xml-base.feature

```diff
@@ -0,0 +1,73 @@
+@no_installroot
+@desctructive
+Feature: microdnf install command on packages with xml:base
+
+@bz1734350
+Scenario: Install packages from local repodata with local packages
```

Comment:
This is a regular package installation test and you're testing various combinations of repository urls, not just `xml:base`, so (unless you decide to split the tests into multiple files - see my other comments), why not just name the file `microdnf/install.feature`?